packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -152,7 +159,21 @@ function WorklistDataGrid(props) {
 
   const getCellData = (cell, cellColumnIndex) => {
     const tabIndex = { tabIndex: '-1' };
-    return props.rowHeaderIndex === cellColumnIndex ? (<th key={cellColumnIndex} {...tabIndex} className={cx('worklist-data-grid-row-header')}>{cell.content}</th>) : (<td key={cellColumnIndex} {...tabIndex} className={cx('worklist-data-grid-cell-data')}>{cell.content}</td>);
+
+    // Determine whether cell is a header or grid cell
+    const WorklistCellTag = props.rowHeaderIndex === cellColumnIndex ? 'th' : 'td';
+
+    return (
+      // Return worklist data grid cell component
+      <WorklistCellTag
+        key={cellColumnIndex}
+        {...tabIndex}
+        className={cx('worklist-data-grid-cell', { masked: cell.isMasked })}
+        aria-label={cell.isMasked ? intl.formatMessage({ id: 'Terra.worklistDataGrid.maskedCell' }) : undefined}
```

Comment:
A consumer should be able to provide a custom message for this in the case that their use of cell masking has a different implicit meaning.

Reply:
Since this acceptance criteria was added so far into the sprint, 7/8, does it make sense to log another JIRA to add this flexibility?  I don't have clear requirements for this.  Would we allow consumers to specify the message for each cell?

Reply:
After talks with Charles we should figure out if this should be on a per-cell basis or per-column. It's also not 100% necessary for MVP so we should make a separate task for it.